latest/ug/ml/node-efa.adoc

```diff
@@ -124,17 +124,116 @@ If you don't have an existing cluster, you can run the following command to crea
 eksctl create cluster -f efa-cluster.yaml
 ----
 +
-NOTE: Because the instance type used in this example has GPUs, `eksctl` automatically installs the NVIDIA Kubernetes device plugin on each instance for you.
-. Deploy the EFA Kubernetes device plugin.
+[NOTE]
+====
+Because the instance type used in this example has GPUs, `eksctl` automatically installs the NVIDIA Kubernetes device plugin on each instance for you when using Amazon Linux 2. This is not necessary for Bottlerocket, as the NVIDIA device plugin is built into Bottlerocket's EKS NVIDIA variant. When `efaEnabled` is set to `true` in the nodegroup configuration, `eksctl` will also automatically deploy the EFA device plugin on the nodes.
+====
+[#efa-bottlerocket]
+=== Using Bottlerocket with EFA
+
+Bottlerocket AMI version 1.28.0 and later include official support for EFA. To use Bottlerocket for EFA-enabled nodes, specify `amiFamily: Bottlerocket` in your configuration. If you need to use a custom AMI ID, you must use standard `nodeGroups` instead of `managedNodeGroups`.
```

Comment:
this heading is showing up as a paragraph and not a heading, I suspect we need to do a syntax check pass on this @chrisnegus @tucktuck9

Reply:
A blank line is needed before the `[#efa-bottlerocket]` line for `=== Using Bottlerocket with EFA` to be printed as a heading. I made the suggestion above.